packages/saltcorn-data/models/table.ts

```diff
@@ -1043,7 +1043,18 @@ class Table implements AbstractTable {
       for (const trigger of triggers) {
         for (const row of rows) {
           // run triggers on delete
-          await trigger.run!(row);
+          if (trigger.configuration?._only_if) {
```

Comment:
move this to a method on Trigger: haltOnOnlyIf 

arguments: row, user
returns true if there is an only_if and it evaluates to falsy.

then you can just `if (trigger.haltOnOnlyIf(row, user)) continue;`

Reply:
Ok